supervisor.go

```diff
@@ -37,14 +38,16 @@ type Supervisor struct {
 	done       chan struct{}
 }
 
-// NewSupervisor create a Supervisor object with supervisor configuration file
-func NewSupervisor(configFile string) *Supervisor {
-	return &Supervisor{
+// NewSupervisor create a Supervisor object with supervisor configuration file and configuration tags.
+func NewSupervisor(configFile string, tags []string) *Supervisor {
+	s := &Supervisor{
```

Comment:
maybe revert back to previous straight return?

Reply:
Done.